src/worker.js

```diff
@@ -54,16 +39,19 @@ create().onRequest('job', ({ contents, type, config, filePath, projectPath, rule
   const configPath = Helpers.getConfigPath(fileDir)
   const relativeFilePath = Helpers.getRelativePath(fileDir, filePath, config)
 
-  const argv = Helpers.getArgv(type, config, rules, relativeFilePath, fileDir, configPath)
+  const cliEngineOptions =
+    Helpers.getCLIEngineOptions(type, config, rules, relativeFilePath, fileDir, configPath)
```

Comment:
That's how I would prefer it as well, but the styling in this project already has issues as far as I'm concerned so I just left it for a future update of the style 😛.